### PR TITLE
Add flag to view the first cert only

### PR DESCRIPTION
**Pull request by prateeknischal**
Add a new flag `--leaf` to view the first certificate only. This switch
is available in `connect` and `dump` and does not affect `--json` or
`--pem` flags and those flags will continue to print all the
certificates. Sample usage:

```
$ certigo connect google.com --leaf
$ certigo dump cert.pem --leaf
```
Please refer https://github.com/square/certigo/pull/178 for history on this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +74,7 @@ Commands:
         --timeout=5s          Timeout for connecting to remote server (can be '5m', '1s', etc).
     -m, --pem                 Write output as PEM blocks instead of human-readable format.
     -j, --json                Write output as machine-readable JSON format.
+	-l, --leaf               Write only the first certificate
```
Comment (mbyczkowski) [inline README.md]: nit: indent

File: README.md [hunk 2]
```diff
@@ -61,6 +61,7 @@ Commands:
     -p, --password=PASSWORD  Password for PKCS12/JCEKS key stores (reads from TTY if missing).
     -m, --pem                Write output as PEM blocks instead of human-readable format.
     -j, --json               Write output as machine-readable JSON format.
+    -l, --leaf               Write only the first certificate
```
Comment (mbyczkowski) [inline README.md]: maybe use `Only display the first certificate` instead?

**Comment by prateeknischal** [comment]
@mbyczkowski , the original idea for the `--depth` flag was to display just supplied number of certs. But, since the mentioned PR, I have moved the `--depth` to only the first, no option. I was thinking about it, for `--json` it does not make a ton of sense because it can be solved using a ` certigo dump file.pem --json | jq ".certificates[0]"`. For the `--pem`, I will be happy to implement it if it feels of use. But for most of the cases, i think only the interactive or visual mode would be useful to have a choice to have `--leaf` flag.

**Comment by mcpherrinm** [comment]
I think if we have `--leaf`, it should work with `--pem` and `--json`, or return an error that they are incompatible.  I think with either of those we'd be good to go.

**Comment by isemaya-square** [comment]
Thank you @prateeknischal for the suggestion. We included the changes in a new PR so that the `--first` flag (we renamed the `--leaf` flag to `--first` since the idea of a leaf doesn't make much sense for cert chains that can be dumped) also works with `--json` and `--pem` flags. https://github.com/square/certigo/pull/256

**Pull request closed without merging**